config/locales/es.yml

```diff
@@ -2,7 +2,7 @@
 es:
   telephony:
     account_reset_cancellation_notice: Votre demande de suppression de votre compte login.gov a été annulée.
```

Comment:
*sigh* will need to update this to be **s**panish, not french.

Reply:
oh. i think i see what the problem is, perhaps a past merge error.

`account_reset_cancellation_notice: Su solicitud para eliminar su cuenta de login.gov ha sido cancelada`

Reply:
Ok. I got it it.